src/lib/entity.js

```diff
@@ -585,20 +586,30 @@ export function printEntity(entity, onDoubleClick) {
  * @return {Element} Entity created
  */
 export function createEntity(definition, cb) {
-  const entity = document.createElement(definition.element);
-
-  // load default attributes
-  for (let attr in definition.components) {
-    entity.setAttribute(attr, definition.components[attr]);
+  let objName = "";
+  while(objName == ""){
+    objName = prompt("Enter the name of the object");
+  }
+
+  if (objName != null){
+    objName += "@" + uuidv4();
+    const entity = document.createElement(definition.element);
+  
+    // load default attributes
```

Comment:
I wonder for our use case, can we by default add the gltf-model attribute here?
Like maybe explore adding on line 603:
```
entity.setAttribute("gltf-model", "");
```
Not sure if it will work, but can you try to see what happens? It may reduce one step for admins, as in admins won't have to manually add in the gltf-component, but instead will just need to select it.